luwak/src/test/java/uk/co/flax/luwak/TestInputDocument.java

```diff
@@ -29,6 +30,17 @@
     public ExpectedException expected = ExpectedException.none();
 
     @Test
+    public void testSetDefaultAnalyzer() {
+
+        InputDocument doc = InputDocument.builder("id")
+                    .setDefaultAnalyzer(new StandardAnalyzer())
+                    .addField("test", "value", new StandardAnalyzer()).build();
```

Comment:
I don't think this is testing what it's supposed to be testing?  You're setting a default analyzer, but then specifying it on the field as well, so the 'use default' code path won't be triggered here.